modules/ROOT/pages/editor-plugin-version.adoc

```diff
@@ -1,160 +1,121 @@
-= Specify editor & plugin versions
-:description_short: Specifying editor and plugin versions for Tiny Cloud deployments.
-:description: Specifying editor and plugin versions for Tiny Cloud deployments.
+= Specify editor & plugins
```

Comment:
This doesn't read right to me. Should the title be `Specify editor version & plugins`?

Reply:
agree

Reply:
```suggestion
= Specify editor version & plugins
```